solr/solrj/src/java/org/apache/solr/client/solrj/impl/ClusterStateProvider.java

```diff
@@ -107,9 +111,15 @@ default DocCollection getCollection(String name) throws IOException {
 
   /** Obtain a cluster property, or the default value if it doesn't exist. */
   default <T> T getClusterProperty(String key, T defaultValue) {
+    Map<String, Object> properties = getClusterProperties();
```

Comment:
The configurable request could now return null, hence the change

Reply:
The change could introduce a regression to a caller, albeit rather minor.  Why not have it return Collections.empty() so the callers can skip the null check?